src/index.html

```diff
@@ -15,8 +15,96 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list__item">
+            <a
+              class="nav__list__item__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              class="nav__list__item__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              class="nav__list__item__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              class="nav__list__item__link"
+              data-qa="hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              class="nav__list__item__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              class="nav__list__item__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              class="nav__list__item__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              class="nav__list__item__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
```

Comment:
Add empty lines between multiline sibling blocks of HTML for better readability. For example, add an empty line between closing `</a>` and next `<li>` tags.